ansible/roles/postldap/defaults/main.yml

```diff
@@ -118,13 +118,23 @@ postldap__vmail_posix_gidnumber: '{{ ansible_local.postldap.vmail_posix_gidnumbe
 # All mailboxes are stored directly in the file system of the server.
 #
 # This setting describes the prefix that the virtual delivery agent prepends to all pathname results from
-# `$virtual_mailbox_maps` table lookups.
+# ``ldap_virtual_mailbox_maps`` table lookups.
```

Comment:
You should revert this change. The `$virtual_mailbox_maps` variable is a Postfix configuration option, not a specific table. The maibox base specified here will be added to the mailbox paths from any lookup table specified here, not just LDAP (think MySQL, or other databases).